tests/system/providers/google/cloud/dataproc/example_dataproc_pig.py

```diff
@@ -31,8 +31,8 @@
 )
 from airflow.utils.trigger_rule import TriggerRule
 
-ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
 DAG_ID = "dataproc_pig"
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
```

Comment:
May I know why do we need to change the order here?

Reply:
> May I know why do we need to change the order here?

In my opinion having DAG_ID on the top is more convenient because it helps to find it faster in the DAG source code and switch to the Arflow UI to find it. It is not a critical enhancement and has no effect on the execution, it just helps me to work with many DAGs in UI faster and more comfortably. Perhaps, it might be good for others too.